### ISSUE TITLE
Proposal: Change (keyup) to (keydown) for most events for JAWS interropability

### ISSUE THREAD

**Issue by jbabbs** [GSA/sam-ui-elements#371]
When running the screen reader JAWS, it does not always capture keyup events. As a result of this, we are unable to select items in the autocomplete component, using the keyboard, when using JAWS. I have yet to see if using keydown fixes this solution, but in any case this is a bug in the autocomplete component when using JAWS (or it could be considered a JAWS bug.

**Comment by thomastighe** [comment]
@jbabbs  Which component are you using?

**Comment by jbabbs** [comment]
We've noticed the issue in <sam-autocomplete>

**Comment by jbabbs** [comment]
The easiest way to reproduce this is to run JAWS and try to select an item from the autocomplete dropdown on https://alpha.sam.gov/release-notes

**Comment by thomastighe** [comment]
@jbabbs  Create a Jira ticket for this so we can work on it and assign to Rinu. I know there were other issues with using Keydown so it may be more than just a slight change. I am moving this code into our new UI. I know I had it reading previously.
